scriptmodules/emulators/dosbox-staging.sh

```diff
@@ -22,24 +22,19 @@ function _get_branch_dosbox-staging() {
 }
 
 function depends_dosbox-staging() {
-    getDepends cmake libasound2-dev libglib2.0-dev libopusfile-dev libpng-dev libsdl2-dev libsdl2-net-dev meson ninja-build
+    getDepends cmake libasound2-dev libglib2.0-dev libopusfile-dev libpng-dev libsdl2-dev libsdl2-net-dev libspeexdsp-dev meson ninja-build
 }
 
 function sources_dosbox-staging() {
     gitPullOrClone
 }
 
 function build_dosbox-staging() {
-    local params=(-Dbuildtype=release -Ddefault_library=static --prefix="$md_inst")
-
-    # Fluidsynth (static)
-    cd "$md_build/contrib/static-fluidsynth"
-    make
-    export PKG_CONFIG_PATH="${md_build}/contrib/static-fluidsynth/fluidsynth/build"
+    local params=(--prefix="$md_inst")
 
     cd "$md_build"
```

Comment:
the `cd` is not needed anymore, was needed because of the separate build step for `fluidsynth`.

Reply:
Nice spot. This one slipped through.